apps/desktop/src/lib/components/AIPromptEdit/DialogBubble.svelte

```diff
@@ -155,8 +155,8 @@
 			background-color var(--transition-fast),
 			border-color var(--transition-fast);
 
-		&:not(.is-error):hover,
-		&:not(.is-error):focus-within {
+		&:not(:global(.is-error)):hover,
+		&:not(:global(.is-error)):focus-within {
```

Comment:
👀

Reply:
Yeah it added a lot of these. I definitley plan to check these, not sure what they're all about yet :thinking: